src/components/SideMenu.tsx

```diff
@@ -0,0 +1,91 @@
+import { useEffect, useState } from 'react';
+import { Events, Link, scroller } from 'react-scroll';
+import { SectionPage } from '../config/menu/model';
+import classnames from 'classnames';
+import { Tab } from '@headlessui/react';
+
+export type SideMenuProps = {
+  id: string;
+  items: SectionPage[];
+  colorMenu?: 'turquoise' | 'white';
+  spy?: boolean;
+};
+
+export const SideMenu = ({ id, items = [], colorMenu = 'turquoise' }: SideMenuProps) => {
+  const [selectedIndex, setSelectedIndex] = useState(0);
+  const classColorMenu = classnames({
+    'bg-turquoise': colorMenu === 'turquoise',
+    'bg-white': colorMenu === 'white',
+  });
+
+  const tabGroupClass = classnames(
+    'justify-around max-h-max',
+    `fixed ${classColorMenu} cursor-pointer`,
+    'top-0 list-none m-0 p-0 w-48 bg-gray-200'
+  );
+
+  const colorTextTabClass = classnames({
+    'text-white': colorMenu === 'turquoise',
+    'text-black': colorMenu === 'white',
+  });
+  const tabClass = classnames(
+    `block text-right ${colorTextTabClass} no-underline p-10 hover:text-dark-violet`
+  );
+
+  const colorBorderLeftClass = classnames({
+    'border-l-white': colorMenu === 'turquoise',
+    'border-l-dark-violet': colorMenu === 'white',
+  });
+
```

Comment:
la gestion des couleurs ici sera a reprendre avec la maj de la palette et l'ajout de l'alias primary notamment

ensuite de maniere generale la gestion des couleurs comme ca, je prefererai passer par ca: `https://github.com/JedWatson/classnames#alternate-bind-version-for-css-modules`

c'est bien plus lisible et fonctionnel